semgrep/semgrep/cli.py

```diff
@@ -102,14 +102,30 @@ def cli() -> None:
         "--include",
         action="append",
         default=[],
-        help="Scan only files or directories that match this pattern; --include='*.jsx' will scan"
-        " the following: foo.jsx, src/foo.jsx, foo.jsx/bar.sh. --include='src' will scan src/foo.py"
-        " as well as a/b/src/c/foo.py. Can add multiple times.",
+        help="Filter files or directories by path. The argument is a pattern"
```

Comment:
According to python docs they are called "glob-style patterns" https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.match idk if we care to add that detail.